app/(protected)/chat/chat-list.tsx

```diff
@@ -0,0 +1,39 @@
+import { FlatList } from "react-native";
+import { Text } from "@ui-kitten/components";
+import ChatThumbnail from "@/components/chat/thumbnail";
+import SkeletonChatThumbnail from "@/components/skeletons/chat-thumbnail";
+
+function ChatList() {
+    const isLoading = true;
+    const chatsThumbnailData = new Array(15).fill(0);
+
+    return (
+        <FlatList
+            data={chatsThumbnailData}
+            showsVerticalScrollIndicator={false}
+            renderItem={(item) =>
+                isLoading ? (
+                    <SkeletonChatThumbnail
+                        key={"skeleton-chat-thumbnail" + item.index}
+                    />
```

Comment:
hmmmm not like this i think, kay if ang data is empty, wala din tong i render. dapat iba siya sa flatlist di siya inside sa renderItem

```tsx
if (isLoading) {
   return <SkeletonChatList /> // Same implementation sa other skeleton nga sila na ang mag flatlist etc etc
}

<FlatList .....> // so tangalin na ang isLoading ternary inside renderItem
```